examples/backends/vllm/mm_router_worker/handler.py

```
@@ -0,0 +1,213 @@
+            logger.warning(f"Indexer query failed: {e}, using first worker")
+            return self.instance_ids[0]
+
+    def _apply_chat_template(self, messages: list[dict]) -> str:
+        """Apply chat template to messages for tokenization."""
+        try:
+            # Try using tokenizer's chat template if available
+            if hasattr(self.tokenizer, "apply_chat_template"):
+                return self.tokenizer.apply_chat_template(
+                    messages, tokenize=False, add_generation_prompt=True
+                )
+        except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)